winner-tool/src/main.rs

```diff
@@ -0,0 +1,179 @@
+//! This tool calculates the quantitative category winners for Tour de SOL.
+//!
+//! NOTE: Ledger processing uses native programs, so this tool must be invoked with `cargo run`.
```

Comment:
Can you please move this info, and other general operating instructions into a new section of https://github.com/solana-labs/tour-de-sol/blob/master/README-internal.md

Reply:
Oh and one better (fine as a follow-up PR if that's easier): a new shell script like https://github.com/solana-labs/tour-de-sol/blob/master/bench-tps.sh that runs the tool with all the right settings against the actual TdS cluster.

Reply:
Great call https://github.com/solana-labs/tour-de-sol/issues/13